maps/CEVEris/_CEV_Eris.dmm

```diff
@@ -32386,43 +32376,13 @@
 /obj/item/stack/material/glass{
 	amount = 120
 	},
-/obj/item/stack/material/glass{
-	amount = 120
-	},
-/obj/item/stack/material/steel{
-	amount = 120
-	},
-/obj/item/stack/material/steel{
-	amount = 120
-	},
-/obj/item/stack/material/steel{
-	amount = 120
-	},
 /obj/item/stack/material/steel{
 	amount = 120
 	},
 /obj/item/stack/material/steel{
 	amount = 120
 	},
-/obj/item/stack/material/steel{
-	amount = 120
-	},
-/obj/item/weapon/reagent_containers/glass/beaker/sulphuric,
```

Comment:
These acids won't pre-loaded in autolathes ?

Reply:
Acids are actually not used by R&D anymore. They were replaced by silicon, and circuit imprinter starts with a 50 unit beaker of silicon.